specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/disk.json

```diff
@@ -741,6 +750,10 @@
         },
         "dataAccessAuthMode": {
           "$ref": "./diskRPCommon.json#/definitions/DataAccessAuthMode"
+        },
+        "optimizedForFrequentAttach": {
+          "type": "boolean",
+          "description": "This property allows you to improve the reliability and performance of your disks that are frequently (greater than 5) moved between VMs by detaching and attaching. This property should be set for disks that are moved frequently (greater than 5 moves) as it can reduce the reliability of the disk otherwise."
```

Comment:
Setting this property to true improves reliability and performance of data disks that are frequently (more than 5 times a day) by detached from one VM and attached to another. This property should not be set for disks that are not detached and attached frequently as it causes the disks to not align with the fault domain of the VM.